### PR TITLE
Model ver fix

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: demos/python_demos/clip_image_classification/download_model_requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
-openvino>=2023.2.0
+openvino==2023.3.0
```
Comment (mzegla) [inline demos/python_demos/clip_image_classification/download_model_requirements.txt:2]: Why is this needed? Now we will need to manually update requirements with every release.

**Review by dkalinowski — APPROVED** [review]

File: demos/python_demos/clip_image_classification/download_model_requirements.txt (hunk 1 above)
Comment (mzegla) [reply demos/python_demos/clip_image_classification/download_model_requirements.txt:2]: Okay, I see it's a PR to the release branch

**Review by mzegla — APPROVED** [review]
